test/unittest/CMakeLists.txt

```diff
@@ -101,6 +103,33 @@ if(is_dpcpp)
   )
 endif()
 
+if(is_dpcpp AND ${TUNING_TARGET} STREQUAL "DEFAULT")
+  string(REGEX MATCH "nvidia_gpu" is_nvidia ${DPCPP_SYCL_TARGET})
+  string(REGEX MATCH "amd_gpu" is_amd ${DPCPP_SYCL_TARGET})
+  if (${DPCPP_SYCL_TARGET} STREQUAL "nvptx64-nvidia-cuda" OR is_nvidia STREQUAL "nvidia_gpu")
```

Comment:
I think this could be simplified to `if (${DPCPP_SYCL_TARGET} MATCHES "nvidia")` and similar for AMD